dae-alright/src/components/formInput/formInput.component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
All components should have their own directory in the `components` folder.

i.e `components/Button/button.js`